src/admin/collectionsOrBundles/views/CollectionOrBundleActualisationOverview.tsx

```diff
@@ -73,8 +73,15 @@ const CollectionOrBundleActualisationOverview: FunctionComponent<CollectionOrBun
 	// computed
 
 	const userGroupOptions = useMemo(
-		() =>
-			userGroups.map(
+		() => [
+			{
+				id: NULL_FILTER,
+				label: t('Geen rol'),
+				checked: get(tableState, 'author.user_groups', [] as string[]).includes(
+					NULL_FILTER
+				),
+			},
```

Comment:
Nuttig om dit object generiek te maken en ergens in `shared` te zetten?

Reply:
kan idd een optimalisatie zijn, maar tot nu to al altijd op deze manier geïmplementeerd dus ik laat het voorlopig zo